src/main/java/ch/njol/skript/events/EvtEffectCommand.java

```diff
@@ -0,0 +1,96 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.events;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.command.EffectCommandEvent;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptEvent;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.ConsoleCommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+public class EvtEffectCommand extends SkriptEvent {
+
+	static {
+		Skript.registerEvent("Effect Command Event", EvtEffectCommand.class, EffectCommandEvent.class,
+				"[sender:(:player|console)] effect command")
+				.description("Called when a player or console executes a skript effect command.")
+				.examples(
+						"on effect command:",
+								"\tlog \"%sender%: %command%\" to file \"effectcommand.log\"")
+				.since("INSERT VERSION");
+	}
+
+	private enum Executor {
+
+		ANY(""),
+		CONSOLE("console "),
+		PLAYER("player ");
```

Comment:
```suggestion
		CONSOLE("console"),
		PLAYER("player");
```
spacing should be handled by the caller.